app/javascript/lib/datetime.js

```diff
@@ -67,3 +67,10 @@ export const parseDay = datetime => {
     weekday: 'long',
   });
 };
+
+export const latestDay = day => {
+  const t = new Date();
+  return new Date(t.setDate(t.getDate() - t.getDay() + day))
+    .toISOString()
+    .slice(0, 10);
+};
```

Comment:
wonder if at this point we should introduce something like `date-fns`

Reply:
we could probably circle back and retro fit this with date-fns right? I can add a ticket for that in the backlog.

Reply:
yea not blocking